ce-samples-post-authn-ui-react/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
`.idea` dir can be excluded as well and removed from git

Reply:
Good catch.  I have added it.